case-lib/hijack.sh

```diff
@@ -8,11 +8,14 @@ function exit()
     local exit_status=${1:-0}
 
     # To generate topology graph, test case should export $tplg for us
-    if [[ -n "$tplg" ]]; then
-        tplgtool.py -d graph -D $LOG_ROOT $tplg &> /dev/null
+    tplg_path=`func_lib_get_tplg_path "$tplg"`
+    if [[ "$?" != "0" ]]; then
```

Comment:
normally `!=` use to compare string, you can find it in some environment setup like:  
`  "X$?" != "X0" ` which is more useful

Reply:
if we quote numbers, "10", it should be string, too